tests/db/test_bigchain_api.py

```diff
@@ -792,20 +807,12 @@ def test_multiple_owners_before_single_owner_after_single_input(self, b, user_sk
         assert len(tx_signed['transaction']['fulfillments']) == 1
         assert len(tx_signed['transaction']['conditions']) == 1
 
+    @pytest.mark.usefixtures('inputs')
     def test_multiple_owners_before_single_owner_after_multiple_inputs(self, b, user_sk, user_vk):
         # create a new users
         user2_sk, user2_vk = crypto.generate_key_pair()
         user3_sk, user3_vk = crypto.generate_key_pair()
 
-        # create inputs to spend
```

Comment:
are you sure you can delete this?

it's a multi-ownership `[user_vk, user2_vk]` instead of single